### PR TITLE
Sprockets rails 3.2.1

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Requires -- https://github.com/theforeman/tfm-ror51-packaging/pull/11

**Comment by ehelms** [comment]
[test copr]

**Comment by ehelms** [comment]
This is ready for review

File: packages/tfm-ror51-rubygem-sprockets-rails/tfm-ror51-rubygem-sprockets-rails.spec [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Summary: Sprockets Rails integration
+Group:   Development/Languages
+License: MIT
+URL:     https://github.com/rails/sprockets-rails
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby >= 1.9.3
+BuildRequires: %{?scl_prefix}rubygem(railties) >= 4.0
```
Comment (ekohl) [inline packages/tfm-ror51-rubygem-sprockets-rails/tfm-ror51-rubygem-sprockets-rails.spec]: This can't be in the same PR since railties is added here.

Comment (ekohl) [reply packages/tfm-ror51-rubygem-sprockets-rails/tfm-ror51-rubygem-sprockets-rails.spec]: Is see you submitted https://github.com/theforeman/tfm-ror51-packaging/pull/14 for that.

**Pull request merged**